melosys-eessi-app/src/main/java/no/nav/melosys/eessi/closebuc/BucLukker.java

```diff
@@ -38,11 +39,11 @@ public void lukkBucerAvType(BucType bucType) {
             //FIXME: søk på BUC fungerer ikke med status open. Venter på eux/rina-fix
```

Comment:
Er fortsatt relevant? Hvordan kan koden fikses hvis dette ikke ligger hos oss?

Reply:
Jepp. Ligger hos eux (eller RINA 🤷 ) 😬  Har purret på dette et par ganger tidligere, men ser ut til at jeg bør purre på nytt igjen siden det ikke var løst (sjekket for et par dager siden)

Reply:
Skjønner 😬 Lurer dog på om vi bør ha kommentaren i koden, fordi vår kode endres vel ikke pga. dette.

Reply:
Jo, hadde egentlig en filtrering på status "open" i BUC-søk-requestet 😄

Reply:
Ja 😄. FIXME bør nok være på linjen som gjelder